app.vue

```diff
@@ -1,6 +1,19 @@
 <template>
-  <div>
+  <div
+    class="prose max-w-none
+    prose-h2:border-t prose-h2:pt-12 prose-h2:scroll-mt-20 prose-h2:lg:scroll-mt-12
```

Comment:
I initially had classes like `prose-h1:type-headline-1` here as this aligns with what is in Figma. I did a review with Junjie and we agreed the text styling given to us by the Tailwind typography plugin is preferred, therefore they have been removed 💅